README.md

```diff
@@ -3,7 +3,7 @@
 maptime.github.io
 =================
 
-This is the central repo of the [maptime.io](http://www.maptime.io/) website. This is a [Jekyll](https://github.com/jekyll/jekyll) site served on Github Pages. If your environment is already setup, type in your terminal (the '$' refers to your prompt):
+This is the central reposetery of the [maptime.io](http://www.maptime.io/) website. This is a [Jekyll](https://github.com/jekyll/jekyll) site served on Github Pages. If your environment is already setup, type in your terminal (the '$' refers to your prompt):
```

Comment:
Hi, 
`reposetery` contains a typo.
Everybody understand repo as repository. Not sure it is more helpful

Reply:
Thanks @shrivastavaashishpro, but agree with @tsamaya that this is not necessary and your PR is a typo.